tracer/src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -701,6 +707,19 @@ private void OnRoutingEndpointMatched(object arg)
             }
         }
 
+        private void ExtractEndpoint(ref Endpoint? endpoint, ref RoutePattern? routePattern, object endpointObj)
```

Comment:
Small suggestion: Since you're just assigning values to the parameters, you could change the two `ref` parameters to `out` parameters. And then you could also move the input object as the first argument to make it a little more idiomatic.

Reply:
Removed in 9e02c49 in the end